### PR TITLE
Cypress/Adding more dispatches to experimental wf and adapting Rancher tests to Rancher 2.7.2

### PR DESCRIPTION
**Pull request by mmartin24**
Adding more dispatches and other adaptations to our e2e tests
 
 ### STD UI Experimental.
 Added dispatches for :
 - Epinio branch selection
 - Helm version
 - Node version

### Rancher UI Experimental
Added:
- Rancher version selector
- Created new script install_rancher_experimental to include PSP enable/disable flag. 
- Logic to enable/disable PSP flag according to the Kubernetes version

### Additional work
- Refactored logic to distinguish between the default Epinio chart (experimental) present in Rancher 2.7.2 and the one uploaded from the repo.
- Refactored where to get the ingresses from in the UI since `Network` was not always available without a refresh
- Refactored locators in menu test `Test buttons and links in dashboard page` to avoid occasional broken tests.
- Removed "falsy" `null` on uninstallation step that produced an incorrect selection of repo installed as in [here](https://github.com/epinio/epinio-end-to-end-tests/actions/runs/4593578988/jobs/8111961195#step:3:90) 

### Tests results:
- [Master Rancher UI EXPERIMENTAL workflow #76](https://github.com/epinio/epinio-end-to-end-tests/actions/runs/4594830751/jobs/8114456084) (`k3s version`: `v1.25.4+k3s1`; `Rancher version`: `2.7.2`, )
- [STD UI experimental template #76](https://github.com/epinio/epinio-end-to-end-tests/actions/runs/4595555076/jobs/8115949767)  (`epinio_branch`: `1726-zero-scale-status`, `helm-version 3.10.2`)
- [Rancher-UI-1-Chrome #539](https://github.com/epinio/epinio-end-to-end-tests/actions/runs/4595226660/jobs/8115219595) > normal one

### CODE REVIEW

File: scripts/install_rancher_experimental.sh [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  --create-namespace \
+  --set installCRDs=true \
+  --set extraArgs[0]=--enable-certificate-owner-ref=true \
+  --wait
+
+# Wait for cert-manager deployment to be ready
+kubectl rollout status deployment cert-manager -n cert-manager --timeout=120s
+
+# Logic for psp enabled according to the Kubernetes version
+KUBERNETES_SERVER_VERSION=$(kubectl version -o json | jq -rj '.serverVersion|.minor' | tr -d '"')
```
Comment (thehejik) [inline scripts/install_rancher_experimental.sh:27]:
```suggestion
KUBERNETES_SERVER_VERSION=$(kubectl version -o json | jq -r '.serverVersion.minor')
```

File: scripts/install_rancher_experimental.sh [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+if  (( $(echo "$KUBERNETES_SERVER_VERSION >= 25" ) )); then
```
Comment (thehejik) [inline scripts/install_rancher_experimental.sh:28]:
```suggestion
if  [ $KUBERNETES_SERVER_VERSION -ge 25 ] ; then
```

File: scripts/install_rancher_experimental.sh (hunk 1 above)
Comment (mmartin24) [reply scripts/install_rancher_experimental.sh:27]: agree. thanks

File: scripts/install_rancher_experimental.sh (hunk 2 above)
Comment (mmartin24) [reply scripts/install_rancher_experimental.sh:28]: agree. thanks.

**Pull request merged**